libafl/src/mutators/mopt_mutator.rs

```diff
@@ -532,6 +533,11 @@ where
             let rand_seed = state.rand_mut().next();
             state.add_metadata::<MOpt>(MOpt::new(mutations.len(), swarm_num, rand_seed)?);
         }
+        let Some(max_stack_pow) = NonZero::new(max_stack_pow) else {
```

Comment:
I'm actually not 100% sure if max_stack_pow of 0 could sometimes make sense(?)
I assumed that not.